kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -925,9 +912,13 @@ public RestMetricsContext getMetricsContext() {
   }
 
   private final class ConfigsBuilder {
-    private final Set<String> mask ;
+    private final Set<String> mask;
     private final Map<String, ConfigValue> configs = new HashMap<>();
 
+    private ConfigsBuilder() {
+      this.mask = null;
```

Comment:
nit: Can we use empty set? I'm nullophobic.

Reply:
You got it.